### PR TITLE
Merge 4.8.2 into 4.9.0

### PR DESCRIPTION
**Pull request by juliamagan**
## Description
This PR carries changes from 4.8.2 into 4.9.0 branch.
### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +43,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Updated ``SQLAlchemy`` and ``urllib3`` in dependencies Azure cloud security section. ([#6842](https://github.com/wazuh/wazuh-documentation/pull/6842))
+- Changed all references to HP-UX package to tar.gz. ([#7107]https://github.com/wazuh/wazuh-documentation/pull/7107)
```
Comment (rauldpm) [inline CHANGELOG.md:46]: Missing () in URL

File: CHANGELOG.md [hunk 2]
```diff
@@ -77,11 +87,14 @@ All notable changes to this project will be documented in this file.
...
 - Fixed password update step for distributed deployments. ([#6967](https://github.com/wazuh/wazuh-documentation/pull/6967))
 - Added fixes and updates to the MITRE ATT&CK framework section. ([#6962](https://github.com/wazuh/wazuh-documentation/pull/6962))
+- Modified master node address name to uppercase. ([#7127](https://github.com/wazuh/wazuh-documentation/pull/7127))
 
 ### Removed
 
 - Deprecated ``/vulnerability`` API endpoints. ([#6738](https://github.com/wazuh/wazuh-documentation/pull/6738))
-- Remove implicit ``id!=000`` WQL filter in the search bar. ([#6815](https://github.com/wazuh/wazuh-documentation/pull/6815))
+- Removed implicit ``id!=000`` WQL filter in the search bar. ([#6815](https://github.com/wazuh/wazuh-documentation/pull/6815))
+- Removed launchtl unload step from macOS uninstalling manual. ([#7123](https://github.com/wazuh/wazuh-documentation/pull/7123))
```
Comment (rauldpm) [inline CHANGELOG.md:96]: `launchtl` or `launchctl`?

It seems that the PR has a typo

File: CHANGELOG.md [hunk 3]
```diff
@@ -77,11 +87,14 @@ All notable changes to this project will be documented in this file.
...
+- Removed Wazuh_rules.pdf references. ([#7142](https://github.com/wazuh/wazuh-documentation/pull/7142))
```
Comment (rauldpm) [inline CHANGELOG.md:97]: It is `Wazuh_Ruleset.pdf`, the PR title has a typo

**Review by rauldpm — CHANGES_REQUESTED** [review]
Minor changes

File: CHANGELOG.md (hunk 1 above)
Comment (juliamagan) [reply CHANGELOG.md]: Done in 0d703e5

File: CHANGELOG.md (hunk 2 above)
Comment (juliamagan) [reply CHANGELOG.md]: Done in 0fb5170

File: CHANGELOG.md (hunk 3 above)
Comment (juliamagan) [reply CHANGELOG.md]: Done in 0fb5170

**Pull request merged**